README.md

```diff
@@ -1 +1,85 @@
-###### official website for the #TechMasters Slack Group [techmasters.chat](http://techmasters.chat)
+# Technology Masters Website
+
+The official website for the #TechMasters Slack Group [techmasters.chat](http://techmasters.chat)
+
+The site is published using [Jekyll](https://jekyllrb.com), a Ruby Gem that creates static
+pages that can be hosted anywhere.
+
+## Installation
+
+The Vagrant installation is recommended because it requires fewer steps, ensures the correct 
```

Comment:
I'll change this to "is the easiest approach" or something else to indicate that it's simpler.